redhat-developer/app-services-cli
# fix: use the OpenShift online terms

### Description
Updates to use OpenShift Online terms

### Verification Steps
1. Create a clean user with access to the service (I can create one if you ask)
2. Run `rhoas kafka create`
3. Confirm that the link generated takes you to a page where you are asked to accept the "Red Hat OpenShift Online Services Agreement"
-->

### Type of change


### Checklist
